modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id="proc-overriding-default-node-ip-selection-logic_{context}"]
```

Comment:
Generally this is identical to the file name itself. Any deviation from this is probably in the oldest of the 4.x files before we were completely consistent.

```
[id="overriding-default-node-ip-selection-logic_{context}"]
```